From: SJ Park <sj@kernel.org>
Date: Wed, 15 Jul 2026 07:15:53 -0700
Subject: [PATCH] ==== attrs monitoring testing ====

mm/damon: add kunit and selftests for probes and probe weights

DAMON recently introduced probes and probe weights.  Add kunit and
selftests for ensuring the parameters for features can be set using the
core API and the sysfs ABI, respectively.

Changes from RFC v1.1
- RFC v1.1: https://lore.kernel.org/20260730043944.146030-1-sj@kernel.org
- None.
Changes from RFC v1
- RFC v1: https://lore.kernel.org/20260729144738.90895-1-sj@kernel.org
- Fix a typo, memory leak and NULL dereference in kunit test.
- Fix ignored error in selftest.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_7uxm_5fs | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_7uxm_5fs

diff --git a/hkml_cv_bogus/hkml_cv_bogus_7uxm_5fs b/hkml_cv_bogus/hkml_cv_bogus_7uxm_5fs
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

